actors/market/src/lib.rs

```diff
@@ -318,7 +319,7 @@ impl Actor {
                 info!("invalid deal: {}: insufficient client funds to cover proposal cost", di);
                 continue;
             }
-            total_provider_lockup += &deal.proposal.provider_collateral;
+            let provider_total_lockup = TokenAmount::zero().add(&deal.proposal.provider_collateral).add(&total_provider_lockup);
             let provider_balance_ok =
                 msm.balance_covered(provider, &total_provider_lockup).map_err(|e| {
```

Comment:
This is using the non-updated variable that doesn't yet include the proposal's provider collateral.  This is a good example of why the names should be more different than flipping two words.  I don't care too much about what you choose but they should be different enough to make mistakes like this more obvious.

Rough ideas: `provider_lockup_for_deal`, `provider_lockup_latest`